doc/source/installation/apt_repo.rst

```diff
@@ -17,8 +17,8 @@ Supported Releases:
 
  * 12.04LTS (precise)
  * 14.04LTS (trusty)
- * 15.10 (wily)
```

Comment:
Maybe move to "formerly supported?" But this would require digging up the whole set of previously supported releases